shell.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs }:
+{ pkgs? import <nixpkgs> {} }:
```

Comment:
This is being explicitly passed in via our flake already right: https://github.com/smartcontractkit/chainlink/blob/develop/flake.nix#L12

What case is this fixing?